web/ui/react-app/src/Panel.tsx

```diff
@@ -161,6 +175,11 @@ class Panel extends Component<PanelProps, PanelState> {
           endTime: endTime,
           resolution: resolution,
         },
+        stats: {
+          loadTime: Date.now() - queryStart,
+          resolution: resolution,
```

Comment:
Just resolution is enough, since key: value have same names, same for totalTimeSeries